src/components/Dropdowns/SelectionDropdown.tsx

```diff
@@ -52,24 +58,34 @@ type SelectionDropdownProps = {
 
 const defaultProps: Partial<SelectionDropdownProps> = {
   showSelectedItemTooltip: true,
+  controlTooltipPlacement: "right",
 };
 
-// Override options in the menu list to include tooltips.
+// Override options in the menu list to include tooltips and, optionally, image content.
 const Option = (props: OptionProps): ReactElement => {
```

Comment:
One more thing: seeing a lot of `as`-casts in this file to assert that the option type is `SelectItem`. I'm also seeing in #528 that `OptionProps` may be generic over its option type? Can you specify that generic here, and if so, does it let you get rid of some of those casts later?

Reply:
Oh! This is a good point, I think I forgot to go back and update `SelectionDropdown` after I added the `SelectItem` type to `StyledSelect`. Yes, I believe I can do `OptionProps<SelectItem>` but I'll update the PR.